_data/people/MohamedAly.yml

```diff
@@ -0,0 +1,10 @@
+active: true
+focus-area:
+  - as
+institution: Princeton University
+name: Mohamed Aly
+photo: /assets/images/team/Mohamed-Aly.png
+shortname: MoAly98
```

Comment:
Not sure why things are failing here, as the `shortname` is valid

```
/home/runner/work/iris-hep.github.io/iris-hep.github.io/_plugins/checks.rb:59:in `raise_err': _data/people/MohamedAly.yml 'shortname' has 'MoAly98' != 'MohamedAly' (expected) 
```

cc @henryiii @rct225

Reply:
@matthewfeickert I changed the yaml file name maybe this fixes it.